src/Indicator.vala

```diff
@@ -86,6 +89,95 @@ public class Network.Indicator : Wingpanel.Indicator {
             popover_widget.closed ();
         }
     }
+
+    private void update_tooltip () {
+        switch (popover_widget.state) {
+            case Network.State.CONNECTING_WIRED:
+                /* If there's only one active ethernet connection,
+                we get back the string "Wired". We won't want to
+                show the user Connecting to "Wired" so we'll have
+                to show them something else if we get back
+                "Wired" from get_active_wired_name () */
+
+                string active_wired_name = get_active_wired_name ();
+
+                if (active_wired_name == "Wired") {
+                    display_widget.tooltip_markup = Granite.markup_accel_tooltip ({}, _("Connecting to wired network"));
```

Comment:
Is there any reason you use markup tooltip? Maybe we can use normal one, like:

```vala
display_widget.tooltip_text = _("Connecting to wired network");
```

Reply:
It was mostly as I was expecting there to be some sort of middle click behaviour like the other indicators and ended up going along with it. I should probably switch it out to just markup tooltip. Thanks!